common/src/main/kotlin/fund/cyber/markets/common/model/Order.kt

```diff
@@ -4,21 +4,28 @@ package fund.cyber.markets.common.model
 import java.math.BigDecimal
 import java.util.*
 
-data class Order (
-        val exchange: String,
-        val pair: TokensPair,
-        val type: OrderType,
-        val timestamp: Long,
-        val epochHour: Long,
-        val orderId: String?,
-        val amount: BigDecimal,
-        val price: BigDecimal
+data class OrderMin(
```

Comment:
Find more suitable name. For example: OrderShort, OrderSummary etc.

Reply:
Renamed to OrderSummary